src/SmartComponents/AdvisoryDetail/AdvisoryDetail.js

```diff
@@ -51,7 +51,7 @@ const AdvisoryDetail = ({ match }) => {
                 breadcrumbs={[
                     {
                         title: intl.formatMessage(messages.titlesPatchAdvisories),
-                        to: paths.advisories.to,
```

Comment:
It could still be preserved that the paths come from `paths`, but it would need to use `path` instead of `to`.